### PR TITLE
Added bubble animation

### PR DESCRIPTION
**Pull request by 09lkei**
Add bubble animation and changed color of bubbles to make the animation a bit more visible

### CODE REVIEW

File: BubbleGame/style.css [hunk 1]
```diff
@@ -73,14 +73,15 @@ body{
 }
 
 .bubble{
-    background-color: #1c29df;
+    background-color: #add8e6;
```
Comment (sarmadhamdani02) [inline BubbleGame/style.css:76]: I think keep the color `#1c4cdf` as it was, it was looking good and also matching with the theme 🤔

File: BubbleGame/style.css [hunk 2]
```diff
@@ -73,14 +73,15 @@ body{
...
     width: 40px;
     height: 40px;
     border-radius: 50%;
     color: #ffffff;
     display: flex;
     align-items: center;
     justify-content: center;
+    animation: bubble-anim 1s infinite;
```
Comment (sarmadhamdani02) [inline BubbleGame/style.css:84]: increase the time to 1.2 or 1.3 seconds.

**Review by sarmadhamdani02 — CHANGES_REQUESTED** [review]

**Comment by 09lkei** [comment]
I think I know how to make that work, I'll work on it shortly👍

**Comment by sarmadhamdani02** [comment]
Sure 👍

**Pull request merged**